intelligent-video-recording/main.py

```diff
@@ -41,7 +41,9 @@ def help():
     if (config_file_name == ""):
         help()
 
-    MainEvntHndlr = QApplication([])
-    MainApp = MainWindow(config_file_name)
-    MainApp.show()
-    MainEvntHndlr.exec()
+    loop=GLib.MainLoop()
+    MainApp = GstDisplay(config_file_name, loop)
+    try:
+        loop.run()
+    except:
+        MainApp.stopPipeline()
```

Comment:
I believe we should stick to `QApplication::exec()` and call `QApplication::exit()` instead. I'm not sure if some Qt related functionality stops working otherwise.